mizar/dp/mizar/workflows/builtins/networkpolicies/create.py

```diff
@@ -103,3 +113,51 @@ def init_data_for_networkpolicy(self):
             "port_table": [],
         }
         return data
+
+    def fill_data_from_directional_traffic_data(self, data, direction, networkpolicy):
+        network_policy_name = networkpolicy["metadata"]["name"]
+        for index, directional_traffic_data in enumerate(networkpolicy["spec"][direction]):
+            indexed_policy_name = "{}_{}_{}".format(network_policy_name, direction, index)
+            if network_policy_name not in data["networkpolicy_map"]:
+                data["networkpolicy_map"][network_policy_name] = set()
+            if indexed_policy_name not in data["networkpolicy_map"][network_policy_name]:
+                data["networkpolicy_map"][network_policy_name].add(indexed_policy_name)
+                data["indexed_policy_count"] += 1
+
+            self.fill_cidrs_from_directional_traffic_data(data, indexed_policy_name, direction, directional_traffic_data)
+
+    def fill_cidrs_from_directional_traffic_data(self, data, indexed_policy_name, direction, directional_traffic_data):
+        if indexed_policy_name not in data["ports_map"]:
+            data["ports_map"][indexed_policy_name] = []
```

Comment:
(optional) data => rules

Reply:
Here data contains both rules and supporting data. So still keep it as data.